Python/pywarpx/picmi.py

```diff
@@ -2383,6 +2383,9 @@ class ParticleDiagnostic(picmistandard.PICMI_ParticleDiagnostic, WarpXDiagnostic
     warpx_format: {plotfile, checkpoint, openpmd, ascent, sensei}, optional
         Diagnostic file format
 
+    warpx_diag_type: {Full, BoundaryScraping, BackTransformed}, 'Full' by default, optional
```

Comment:
Hm, allowing `BackTransformed` makes it confusing, since this seems to duplicate the capability of `LabFrameParticleDiagnostic`... We can discuss this in more detail.